merger.go

```diff
@@ -79,7 +79,7 @@ func (m *merger) Run() {
 func (m *merger) computeSpeed() {
 	now := time.Now().UnixNano()
 
-	if now - m.lastUpdate >= int64(time.Second) {
+	if now-m.lastUpdate >= int64(time.Second) {
```

Comment:
![This change is bad, and you should feel bad](https://i.imgflip.com/2fhwnr.jpg)

Reply:
Not me, `gofmt` is to blame :D